libraries/Microsoft.BotKit/Microsoft.BotKit.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Same comment - as these are in our primary build, please make this match our existing project files in terms of versioning and the like.